exporter/convert/message.go

```diff
@@ -29,13 +29,13 @@ func (msg MessageID) GetDutyExecutorID() []byte {
 
 func (msg MessageID) GetRoleType() RunnerRole {
 	roleByts := msg[roleTypeStartPos : roleTypeStartPos+roleTypeSize]
-	return RunnerRole(binary.LittleEndian.Uint32(roleByts))
+	return RunnerRole(binary.LittleEndian.Uint32(roleByts)) // #nosec G115
```

Comment:
if we're using `nosec` here, I think a short explanation of why there's no risk is required.